load.php

```diff
@@ -197,7 +197,8 @@ function perflab_is_valid_module( $module ) {
 	}
 
 	// Do not load module if it cannot be loaded, e.g. if it was already merged and is available in WordPress core.
-	return perflab_can_load_module( $module );
+	$can_load_module = perflab_can_load_module( $module );
+	return is_wp_error( $can_load_module ) ? false : $can_load_module;
```

Comment:
Nitpick, this seems more readable:

```suggestion
	return $can_load_module && ! is_wp_error( $can_load_module );
```